frontend/src/features/auth/auth-enhancements.css

```
@@ -0,0 +1,75 @@
+ * .auth-message-stack still supplies the --auth-message-* tokens, so the banner
+ * card design is unchanged.
+ *
+ * Moved out of an inline <style> (templates/account/snippets/auth_toast.html)
+ * into the auth Vite entry so it ships as one cached asset.
+ */
+.auth-message-toast {
+	align-items: flex-end !important;
+	background: transparent !important;
+	bottom: auto !important;
+	gap: 10px;
+	inset: auto !important;
```

Stylelint (17.12.0)

[error] Overridden property "bottom" by shorthand "inset" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)